api/meter_test.go

```diff
@@ -11,10 +11,12 @@ import (
 	"github.com/google/go-cmp/cmp"
 	logger "github.com/sirupsen/logrus"
 
-	"github.com/pokt-foundation/portal-api-go/repository"
+	"github.com/pokt-foundation/portal-db/types"
 	"github.com/pokt-foundation/utils-go/numbers"
 )
 
+var testCtx = context.Background()
```

Comment:
Is there a specific reason this is defined once and shared between all test functions?  If not, it is best for each function to define and reuse its own local instance of Context object.

Reply:
fixed